<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QDialog" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>1111</width>
     <height>740</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>TABLE</string>
   </property>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>680</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>881</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Tables in the Garden</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>821</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="g2_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="g1_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="g3_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="g4_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="g5_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="g1_pushButton">
          <property name="text">
           <string>Garden1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="g2_pushButton">
          <property name="text">
           <string>Garden2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="g3_pushButton">
          <property name="text">
           <string>Garden3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="g4_pushButton">
          <property name="text">
           <string>Garden4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="g5_pushButton">
          <property name="text">
           <string>Garden5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>881</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Tables in the Terrace</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>821</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="t1_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="t2_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="t3_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="t4_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="t5_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="t1_pushButton">
          <property name="text">
           <string>Terrace1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="t2_pushButton">
          <property name="text">
           <string>Terrace2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="t3_pushButton">
          <property name="text">
           <string>Terrace3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="t4_pushButton">
          <property name="text">
           <string>Terrace4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="t5_pushButton">
          <property name="text">
           <string>Terrace5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>881</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Tables in the Saloon</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>821</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="s1_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s2_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s3_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s4_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="s5_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="s1_pushButton">
          <property name="text">
           <string>Saloon1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="s2_pushButton">
          <property name="text">
           <string>Saloon2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="s3_pushButton">
          <property name="text">
           <string>Saloon3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="s4_pushButton">
          <property name="text">
           <string>Saloon4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="s5_pushButton">
          <property name="text">
           <string>Saloon5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
